hodl/exchanges/PoloniexHelperAPI.py

```diff
@@ -51,8 +46,6 @@ def print_poloniex_ascii():
           '\_|  \___/|_|\___/|_| |_|_|\___/_/\_\\')
 
 def print_poloniex_trade_history():
-    if settings.poloniex == 'on':
-
         start_date = datetime.date(2015, 1, 1)
         end_date = datetime.date.today()
```

Comment:
Since you removed the conditional statement here you will want to un-indent these two lines and any other logic (below) contained inside `print_poloniex_trade_history()`.

Indentation is absolutely critical to how code functions in python. Other languages such as java and c++ use `{}` brackets to encapsulate methods and code but python uses blank space and indentation making the position of things critical to proper function of your code. More about this can be read in the PEP8 guidelines for Python here: https://legacy.python.org/dev/peps/pep-0008/#indentation